birdhouse/default.env

```diff
@@ -84,19 +84,14 @@ export SERVER_LICENSE_URL='${__DEFAULT__SERVER_LICENSE_URL}'
 # Those will not be set explicitly as defaults to ensure they are overridden explicitly by the instance.
 # These values would be detected only if the instance was configured using a copy of 'env.local.example'.
 export __DEFAULT__MAGPIE_SECRET=itzaseekrit
-#export __DEFAULT__MAGPIE_ADMIN_USERNAME=admin
 export __DEFAULT__MAGPIE_ADMIN_PASSWORD=qwertyqwerty!
-#export __DEFAULT__POSTGRES_PAVICS_USERNAME=postgres-pavics
 export __DEFAULT__POSTGRES_PAVICS_PASSWORD=postgres-qwerty
-#export __DEFAULT__POSTGRES_MAGPIE_USERNAME=postgres-magpie
```

Comment:
These might be relevant to warn as well. They should be used everywhere in the configs. 
Since this is for database endpoint connection rather than Magpie user profile, it should not be involved in other scripts than direct interactions between Magpie/Twitcher and Postgres.